libnavigation-ui/build.gradle

```diff
@@ -0,0 +1,111 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'com.jaredsburrows.license'
+apply plugin: 'com.mapbox.android.sdk.versions'
+apply from: "${rootDir}/gradle/ktlint.gradle"
+
+android {
+  compileSdkVersion androidVersions.compileSdkVersion
+  buildToolsVersion androidVersions.buildToolsVersion
+
+  defaultConfig {
+    minSdkVersion androidVersions.minSdkVersion
+    targetSdkVersion androidVersions.targetSdkVersion
+    testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    vectorDrawables.useSupportLibrary = true
+
+    buildConfigField('String',
+        "MAPBOX_NAVIGATION_SDK_IDENTIFIER", String.format("\"%s\"", "mapbox-navigation-ui-android")
+    )
+    buildConfigField 'String',
+        "MAPBOX_NAVIGATION_EVENTS_USER_AGENT", String.format("\"mapbox-navigation-ui-android/%s\"",
+        project.VERSION_NAME
+    )
+    consumerProguardFiles 'proguard-consumer.pro'
+  }
+
+  configurations {
+    javadocDeps
+  }
+
+  buildTypes {
+    debug {
+      testCoverageEnabled = true
+    }
+  }
+
+  testOptions {
+    unitTests.returnDefaultValues = true
+    unitTests.includeAndroidResources = true
+    unitTests.all {
+      maxHeapSize = "1024m"
+      jacoco {
+        includeNoLocationClasses = true
+      }
+    }
+  }
+
+  licenseReport {
+    generateHtmlReport = false
+    generateJsonReport = true
+    copyHtmlReportToAssets = false
+    copyJsonReportToAssets = false
+  }
+}
+
+dependencies {
+  //ktlint
+  ktlint dependenciesList.ktlint
+
+  // Navigation SDK
+  api project(':libnavigation-core')
+  api project(':libnavigation-base')
```

Comment:
https://github.com/mapbox/mapbox-navigation-android/pull/2464#discussion_r381469109

Reply:
This is not needed as we are doing `api project(':libnavigation-core')` which already has the `base`